src/styles/tools/_mixins.scss

```diff
@@ -69,3 +76,14 @@
     margin-top: #{$margin}px;
   }
 }
+
+// Create selectors to allow paragraphs to evenly separate.
+// ---
+// @param {number} $font-size: A font-size
+// @content: A set of selectors and modifiers for altering the visual layout
+//  of paragraph elements
+@mixin create_paragraph_spacing($margin) {
+  & + & {
```

Comment:
what does `& + &` do again? worth a comment?

Reply:
Added a comment. It's a mixin that will generate sibling selectors based on its parent selector.